recipes/adwaita-icon-theme/meta.yaml

```diff
@@ -0,0 +1,69 @@
+{% set name = "adwaita-icon-theme" %}
+{% set version = "3.36.0" %}
+{% set majmin_version = "3.36" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://download.gnome.org/sources/{{ name }}/{{ majmin_version }}/{{ name }}-{{ version }}.tar.xz
+  sha256: 1a172112b6da482d3be3de6a0c1c1762886e61e12b4315ae1aae9b69da1ed518
+
+build:
+  number: 0
+  merge_build_host: False
+  ignore_run_exports:
+    - gtk3  # [win]
+    - librsvg  # [win]
+
+requirements:
+  build:
+    # gtk3 and librsvg here for rendering svg to png, build ok because platform independent
+    - gtk3  # [unix]
+    - librsvg  # [unix]
```

Comment:
Why are these needed in `build` ?

Reply:
They are needed for rendering some SVG icons to PNGs and for building the icon theme cache. I suppose they could go in host if that's what you're asking, but that function seemed more like a platform-independent build tool to me. Practically, we also don't want them as run dependencies, so if they are moved to host then ignore_run_exports will need to be set as with the Windows build.